includes/snmp.inc.php

```diff
@@ -305,6 +305,10 @@ function snmp_get($device, $oid, $options = null, $mib = null, $mibdir = null)
     $measure->manager()->recordSnmp($measure->end());
     if (preg_match('/(No Such Instance|No Such Object|No more variables left|Authentication failure)/i', $data)) {
         return false;
+    } elseif (preg_match('/Wrong Type(.*)should be/', $data)) {
```

Comment:
This should probably be a str_starts_with() instead

Reply:
I just copied https://github.com/librenms/librenms/blob/master/includes/snmp.inc.php#L424